src/content/docs/plugin/updater.mdx

```diff
@@ -9,9 +9,285 @@ sidebar:
+import CommandTabs from '@components/CommandTabs.astro';
+import { TabItem, Steps, Tabs } from '@astrojs/starlight/components';
 
 <PluginLinks plugin="updater" />
 
-<Stub>
-  Based on https://github.com/tauri-apps/plugins-workspace/tree/plugins/updater
-</Stub>
+Automatically update your Tauri app with an update server or a static JSON.
+
+## Supported Platforms
+
+- Windows
+- Linux
+- macOS
+
+## Setup
+
+Install the Tauri updater plugin to get started.
+
+<Tabs>
+  <TabItem label="Automatic">
+
+    Use your project's package manager to add the dependency:
+
+    {' '}
+
+    <CommandTabs
+      npm="npm run tauri add updater"
+      yarn="yarn run tauri add updater"
+      pnpm="pnpm tauri add updater"
+      cargo="cargo tauri add updater"
+    />
+
+  </TabItem>
+    <TabItem label="Manual">
+      <Steps>
+
+        1. Run `cargo add tauri-plugin-updater` to add the plugin to the project's dependencies in `Cargo.toml`.
+
+        2.  Modify `lib.rs` to initialize the plugin:
+
+            ```rust title="lib.rs" ins={4}
+            #[cfg_attr(mobile, tauri::mobile_entry_point)]
+            fn run() {
+                tauri::Builder::default()
+                    .plugin(tauri_plugin_updater::Builder::new().build())
+                    .run(tauri::generate_context!())
+                    .expect("error while running tauri application");
+            }
+            ```
+
+        3.  You can install the JavaScript Guest bindings using your preferred JavaScript package manager:
+
+            <CommandTabs
+                npm="npm install @tauri-apps/plugin-updater"
+                yarn="yarn add @tauri-apps/plugin-updater"
+                pnpm="pnpm add @tauri-apps/plugin-updater"
+            />
+
+      </Steps>
+    </TabItem>
+
+</Tabs>
+
+## Signing updates
+
+Tauri's updater needs a signature to verify that the update is from a trusted source.
+
+To sign your updates you need two things:
+
+1. The public key, which will be set in the `tauri.config.json` to validate the artifacts before the installation.
+2. The private key, which is used to sign your installer files. You should NEVER share this key with anyone. Also, if you lose this key you will NOT be able to publish new updates to the users that have the app already installed. It is important to store this key in a safe place!
+
+To generate the keys on Linux and macOS you can use the Tauri CLI:
+
+<Tabs>
+  <TabItem label="npm">
+  ```zsh
+  npm run tauri signer generate -- -w ~/.tauri/myapp.key
+  ```
+  </TabItem>
+  <TabItem label="yarn">
+  ```zsh
+  yarn tauri signer generate -w ~/.tauri/myapp.key
+  ```
+  </TabItem>
+  <TabItem label="pnpm">
+  ```zsh
+  pnpm tauri signer generate -w ~/.tauri/myapp.key
+  ```
+  </TabItem>
+  <TabItem label="bun">
+  ```zsh
+  bunx tauri signer generate -w ~/.tauri/myapp.key
+  ```
+  </TabItem>
+  <TabItem label="cargo">
+  ```zsh
+  cargo tauri signer generate -w ~/.tauri/myapp.key
+  ```
+  </TabItem>
+</Tabs>
```

Comment:
could you replace these with command tabs, you can place bun too, right now the component doesn't have support but there's a PR to fix that.